conf/paths.go

```diff
@@ -12,6 +12,7 @@
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
 
+//go:build !local
```

Comment:
Where are these comments coming from?

Reply:
So, appareantly this is coming with `Go 1.17`, and is the new way of adding tags, so this is a `gofmt` thing.

So no need to fear, but still, removing them from this